Dockerfile.alpine

```diff
@@ -0,0 +1,42 @@
+FROM python:3.7.3-alpine3.9 AS build
+
+ARG DOCKER_COMPOSE_GITSHA=unknown
+ENV DOCKER_COMPOSE_GITSHA=$DOCKER_COMPOSE_GITSHA
+
+ENV BUILD_BOOTLOADER=1
```

Comment:
These `ARG` and `ENV` lines don't influence the `apk add ...`, and are not needed until further down (or even at runtime?); move them lower in the Dockerfile to prevent busting the cache if not needed

Reply:
Done